server/client_server.go

```diff
@@ -22,10 +24,122 @@ func newServerClient(jiraClient *jira.Client) Client {
 	}
 }
 
+type IssueInfo struct {
+	Values []*jira.MetaIssueType `json:"values,omitempty"`
+}
+
+type FieldInfo struct {
+	Values []interface{} `json:"values,omitempty"`
+}
+
+type FieldValues struct {
+	FieldID string `json:"fieldId,omitempty"`
+}
+
+type FieldID struct {
+	Values []FieldValues `json:"values,omitempty"`
+}
+
+type Version struct {
+	VersionInfo string `json:"version,omitempty"`
+}
+
+const (
+	CreateMetaAPIEndpoint = "rest/api/2/issue/createmeta/"
+	PivotVersion          = "8.4.0"
+)
+
 // GetCreateMeta returns the metadata needed to implement the UI and validation of
 // creating new Jira issues.
 func (client jiraServerClient) GetCreateMeta(options *jira.GetQueryOptions) (*jira.CreateMetaInfo, error) {
```

Comment:
add a new line before this function
```suggestion
func (client jiraServerClient) GetCreateMeta(options *jira.GetQueryOptions) (*jira.CreateMetaInfo, error) {
```